src/base/preferences.cpp

```diff
@@ -62,6 +62,8 @@
 
 Preferences* Preferences::m_instance = 0;
 
+static const QString LOG_FOLDER("logs");
```

Comment:
Just want to note that the usage of `static` is deprecated for this purpose by the C++11 standard. Since the code is new, perhaps is might make sense to use the anonymous namespace.